Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -347,17 +347,6 @@ def exchangelib_cleanup():     # pragma: no cover
         exchangelib.close_connections()
     except Exception as ex:
         demisto.error("Error was found in exchangelib cleanup, ignoring: {}".format(ex))
-    for key, protocol in key_protocols:
-        try:
-            if "thread_pool" in protocol.__dict__:
-                demisto.debug('terminating thread pool key{} id: {}'.format(key, id(protocol.thread_pool)))
-                protocol.thread_pool.terminate()
-                del protocol.__dict__["thread_pool"]
-            else:
-                demisto.info(
-                    'Thread pool not found (ignoring terminate) in protcol dict: {}'.format(dir(protocol.__dict__)))
-        except Exception as ex:
-            demisto.error("Error with thread_pool.terminate, ignoring: {}".format(ex))
```

Comment:
Why is this being removed?

Reply:
this is the famous error print thats been printed everywhere in the last year+ without having an affect
Every time this runs it fails on a type error, so it never actually can do anything
Its also redundant, the point here seems that we didnt trust shut down to do its job for some reason and we are manually trying to do this here (without success not that anybody has noticed)